setups/fullnode/systemd/setup.sh

```diff
@@ -162,29 +178,31 @@ fi
 # Move bin to $BIN_DIR
 cp ./target/release/iota-node "$BIN_DIR/iota-node"
 
-EXEC_START_CMD="\"$BIN_DIR/iota-node\" --config-path \"$CONFIG_DIR/fullnode.yaml\""
-
-# Create a systemd service definition file
-SERVICE_DEF=$(
-	cat "$CLONE_DIR/setups/fullnode/systemd/iota-node.service" |
-		# Set the start command to use your paths to the iota-node binary / to the config file
-		sed "s|/usr/local/bin/iota-node --config-path /opt/iota/config/validator.yaml|$EXEC_START_CMD|g"
-)
-write_to_file "$SERVICE_DEF" "/etc/systemd/system/iota-node.service"
-
-# Files might have been created / overwritten by root user
-sudo chown -R iota:iota "$NODE_WORKDIR"
-sudo chown -R iota:iota "$BIN_DIR"
-sudo chown -R iota:iota "$CONFIG_DIR"
-# Reload systemd with this new service unit file
-sudo systemctl daemon-reload
-# Enable the new service with systemd
-sudo systemctl enable iota-node.service
-# Start the Validator
-sudo systemctl start iota-node
-
-# Wait to catch start failures more efficiently
-sleep 1s
-
-# Check that the node is up and running
-sudo systemctl status --no-pager --no-legend iota-node
+# Setup systemd service (Linux only)
+if [ "$(uname -s)" == "Linux" ]; then
```

Comment:
So what is the idea on Mac? User can start the process himself?

Reply:
ah, I saw in the docs. But maybe we should print a message in that case, so the user directly knows that he has to start it himself?